source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+     --o-table table-dada2.qza \
+     --o-denoising-stats stats-dada2.qza
+
+.. command-block::
+
+   qiime metadata tabulate \
+     --m-input-file stats-dada2.qza \
+     --o-visualization stats-dada2.qzv
+
+If you'd like to continue the tutorial using this FeatureTable (opposed to the Deblur feature table generated in *Option 2*), run the following commands.
+
+.. command-block::
+
+   mv rep-seqs-dada2.qza rep-seqs.qza
+   mv table-dada2.qza table.qza
+
+Option 2: Deblur
+~~~~~~~~~~~~~~~~
+
+`Deblur`_ uses sequence error profiles to associate erroneous sequence reads with the true biological sequence from which they are derived, resulting in high quality sequence variant data. This is applied in two steps. First, an initial quality filtering process based on quality scores is applied. This method is an implementation of the quality filtering approach described by `Bokulich et al. (2013)`_.
+
+.. command-block::
+
+   qiime quality-filter q-score \
+    --i-demux demux.qza \
+    --o-filtered-sequences demux-filtered.qza \
+    --o-filter-stats demux-filter-stats.qza
+
+.. note:: In the `Deblur`_ paper, the authors used different quality-filtering parameters than what `they currently recommend after additional analysis <https://qiita.ucsd.edu/static/doc/html/deblur_quality.html>`_. The parameters used here are based on those more recent recommendations.
+
+Next, the Deblur workflow is applied using the ``qiime deblur denoise-16S`` method. This method requires one parameter that is used in quality filtering, ``--p-trim-length n`` which truncates the sequences at position ``n``. In general, the Deblur developers recommend setting this value to a length where the median quality score begins to drop too low. On these data, the quality plots (prior to quality filtering) suggest a reasonable choice is in the 115 to 130 sequence position range. This is a subjective assessment. One situation where you might deviate from that recommendation is when performing a meta-analysis across multiple sequencing runs. In this type of meta-analysis, it is critical that the read lengths be the same for all of the sequencing runs being compared to avoid introducing a study-specific bias. Since we already using a trim length of 120 for ``qiime dada2 denoise-single``, and since 120 is reasonable given the quality plots, we'll pass ``--p-trim-length 120``. This next command may take up to 10 minutes to run.
+
+.. command-block::
+
+   qiime deblur denoise-16S \
+     --i-demultiplexed-seqs demux-filtered.qza \
+     --p-trim-length 140 \
+     --o-representative-sequences rep-seqs-deblur.qza \
+     --o-table table-deblur.qza \
+     --p-sample-stats \
+     --o-stats deblur-stats.qza
+
+.. note:: The two commands used in this section generate QIIME 2 artifacts containing summary statistics. To view those summary statistics, you can visualize them using ``qiime metadata tabulate`` and ``qiime deblur visualize-stats``, respectively:
```

Comment:
This seems more appropriate:

The two commands used in this section generate --> The deblur command used above generates